packages/integration-test/babel.config.js

```diff
@@ -1,4 +0,0 @@
-module.exports = {
-  presets: [['@babel/preset-env', { targets: { node: 'current' } }]],
-  plugins: ['@babel/plugin-transform-runtime', '@babel/plugin-transform-modules-commonjs'],
```

Comment:
@aimensahnoun not needed anymore, thanks to https://github.com/RequestNetwork/requestNetwork/pull/1605 🎉